src/components/ButtonsLanguages.jsx

```diff
@@ -1,6 +1,19 @@
 import style from './ButtonLanguage.module.css';
+import { useCharacteristicsContext } from '../hooks/useCharacteristicsContext';
 
-function ButtonLanguage({ children }) {
-  return <button className={style.btn}>{children}</button>;
+function ButtonLanguage({ children, close }) {
+  const { handleSelect, handleRemove } = useCharacteristicsContext();
+  return (
+    <>
+      <p onClick={() => handleSelect(children)} className={style.btn}>
+        {children}
+        {close ? (
+          <button onClick={() => handleRemove(children)} className={style.close}></button>
+        ) : (
+          ''
+        )}
+      </p>
+    </>
```

Comment:
React.Fragment is not needed here.

Reply:
And the name of component is Button, but you chose a p tag for it. Why?

Semantically p isn't the right element to handle clicks.

Reply:
for naw from what component we rendered our buttons :)

Reply:
Is it a statement or a question?

Reply:
I just changed it to parentName